<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_Result</class>
 <widget class="QWidget" name="frm_Result">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Order</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/user.png</normaloff>resources/user.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>153</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'å¾®è½¯éé»'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;left&quot; style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600; color:#008000;&quot;&gt;Order list:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_TotalPrice">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>332</y>
     <width>301</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h3 align=&quot;left&quot;&gt;
    &lt;font color=&quot;green&quot;&gt;The total price is :&lt;/font&gt;
    &lt;font color=&quot;red&quot;&gt;0&lt;/font&gt;
    &lt;font color=&quot;green&quot;&gt;$&lt;/font&gt;
&lt;/h3&gt; 
</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>47</y>
     <width>581</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Return">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>357</y>
     <width>581</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/cross.png</normaloff>resources/cross.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_SubmitTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
